app/src/main/res/layout/activity_home_.xml

```diff
@@ -379,66 +379,123 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 app:columnCount="2"
-                app:rowCount="3"
+                app:rowCount="2"
                 app:useDefaultMargins="true">
 
-                <LinearLayout
```

Comment:
Make a custom view for the tile so that we don't have to code every attribute for every tile independently.